src/components/schedule-page/validation-drawer/error-list.component.tsx

```diff
@@ -17,7 +17,10 @@ export default function ErrorList({ errors = [] }: Options): JSX.Element {
         {errors?.length > 0 &&
           errors.map(
             (error, index): JSX.Element => (
-              <ErrorListItem key={error.message.substr(2, 9) + index} error={error} />
+              <ErrorListItem
+                key={(error.message ? error.message.substr(2, 9) : "0") + index}
```

Comment:
Zmieniłam, bo `message` nie jest wymagana dla wszystkich błędów [chociaż skoro już doszły do tego momentu, to powinny mieć do tego pola przypisaną jakąś wartość] - ale warningi się upominały.